### ISSUE TITLE
Timeline invalidating on repeat not working since 1.18.1

### ISSUE THREAD

**Issue by GillesVermeulen** [greensock/GreenSock-JS#175]
We used a timeline that repeats indefinitely, and on each repeat, we invalidate the timeline, so that the object that is being tweened keeps moving in the same direction, instead of resetting every time.

This is the code that worked up until 1.18.0:

``` javascript
this.characterTimeline = new TimelineMax({
        repeat: -1,
        paused: false,
        onRepeat: function() {
          this.characterTimeline.invalidate()
        },
        onRepeatScope: this
});

this.characterTimeline.to(this.character.position, 2, {x: "+=200", ease: Quad.easeOut});
```

Since 1.18.1, this does not work anymore. The character stops, the timeline stops repeating and just seems to be killed off.

**Comment by jackdoyle** [comment]
I see what you mean. Let me look into this and get back to you.

**Comment by jackdoyle** [comment]
I'm so sorry about the delay - I actually fixed this a while back but forgot to post back here. My bad. I believe this is resolved in the upcoming release of 1.19.1 which you can preview (uncompressed) at https://s3-us-west-2.amazonaws.com/s.cdpn.io/16327/TweenMax-latest-beta.js (TweenMax contains TimelineMax). Does that work better for you?
